modules/ROOT/pages/to-set-permissions.adoc

```diff
@@ -4,35 +4,37 @@ include::_attributes.adoc[]
 
-== Exchange Role Groups
+== Exchange Permissions
 
-These role groups pertain to every Business Group level:
+Exchange permissions are the best way to control access to assets in a business group. Before Exchange permissions were introduced, very similar Exchange role groups controlled Exchange access, and role groups are still supported.
 
-* Exchange Viewers: This role group gives permission to users to view and download assets within a business group. The users with this role group cannot add new assets, edit asset portal content, or share an asset with another user. Assign this role to those who consume your assets in a specific business group.
+These permissions pertain to every Business Group level:
 
-* Exchange Contributors: This role group gives permission to users to view, create, and download assets within a business group. Users with this role group can edit asset portal content in an existing asset version. Use this role to invite users to edit and maintain your asset portal descriptions.
+* Exchange Viewers: This permission enables users to view and download assets within a business group. The users with this permission cannot add new assets, edit asset portal content, or share an asset with another user. Assign this permission to those who consume your assets in a specific business group.
 
-* Exchange Administrators: This role group gives permission to users to view, create, and download assets within a business group. Users with this role group can do everything that the Exchange Contributors and Exchange Viewers roles can do plus share an asset with another user, deprecate an asset, and delete an asset. Use this role to set up Exchange Administrators for all the assets within a specific business group.
+* Exchange Contributors: This permission enables users to view, create, and download assets within a business group. Users with this permission can edit asset portal content in an existing asset version. Use this permission to invite users to edit and maintain your asset portal descriptions.
 
-== Asset Roles
+* Exchange Administrators: This permission enables users to view, create, and download assets within a business group. Users with this permission have all the same access as users with the Exchange contributor and Exchange viewer permissions, and access to share an asset with another user, deprecate an asset, and delete an asset. Use this permission to set up Exchange administrators for all the assets within a specific business group.
 
-These role groups pertain to every asset level:
+== Asset Permissions
 
-These roles extend access on assets in addition to the role group that a user has at the business group level:
+Asset permissions are the best way to control access to individual assets. Before asset permissions were introduced, very similar asset role groups controlled asset access, and role groups are still supported.
```

Comment:
```suggestion
Use asset permissions to control access to assets in a business group. You can assign asset permissions to individual users or by using the xref:access-management::teams.adoc[Teams] feature.
```

Reply:
I *think* you can use teams for this? Please verify.

Reply:
Fixed.